docs/content/configuration/index.md

```diff
@@ -24,6 +24,7 @@ Many of Druid's external dependencies can be plugged in as modules. Extensions c
 |`druid.extensions.directory`|The root extension directory where user can put extensions related files. Druid will load extensions stored under this directory.|`extensions` (This is a relative path to Druid's working directory)|
 |`druid.extensions.hadoopDependenciesDir`|The root hadoop dependencies directory where user can put hadoop related dependencies files. Druid will load the dependencies based on the hadoop coordinate specified in the hadoop index task.|`hadoop-dependencies` (This is a relative path to Druid's working directory|
 |`druid.extensions.loadList`|A JSON array of extensions to load from extension directories by Druid. If it is not specified, its value will be `null` and Druid will load all the extensions under `druid.extensions.directory`. If its value is empty list `[]`, then no extensions will be loaded at all. It is also allowed to specify absolute path of other custom extensions not stored in the common extensions directory.|null|
+|`druid.extensions.excludeModules`|A JSON array of canonical class names (e. g. `"io.druid.somepackage.SomeClass"`) of modules, which shouldn't be loaded, despite they are found in the `loadList`. Useful when some useful extension contains some module, which shouldn't be loaded on some Druid node type because of some dependencies of that module couldn't be satisfied.|[]|
```

Comment:
Ah, I see. In that case, could you please change the wording of of this part:

> modules, which shouldn't be loaded, despite they are found in the `loadList`

It makes it sound like the things in moduleExcludeList are the same kind of things as found in loadList. That's what confused me. Perhaps wording like this would be clearer.

> A JSON array of canonical class names (e. g. `"io.druid.somepackage.SomeClass"`) of module classes which shouldn't be loaded, even if they are found in extensions specified by `druid.extensions.loadList`.